docs/notebooks/kurucz_chianti_h_he.ipynb

```diff
@@ -1,696 +0,0 @@
-{
```

Comment:
Does this still work with the current code version? If yes, we might consider keeping it for now since we cannot yet create the standard Tardis atomic data with the new version.

Reply:
Yes, it works. I've updated it because have some typos and wasn't fully working.